polaris-react/src/components/MediaCard/MediaCard.tsx

```diff
@@ -156,8 +157,17 @@ export function MediaCard({
         <div className={mediaContainerClassName}>{children}</div>
         <div className={infoContainerClassName}>
           <LegacyCard.Section>
-            {popoverActionsMarkup}
-            {popoverActionsMarkup ? null : <div className={styles.Popover}>{dismissButtonMarkup}</div>}
+            <Box
+              position="absolute"
+              insetBlockStart="4"
+              insetInlineEnd="5"
+              zIndex="var(--p-z-index-2)"
+            >
+              <Inline gap="1">
+                {popoverActionsMarkup}
+                {dismissButtonMarkup}
+              </Inline>
+            </Box>
```

Comment:
if neither `popoverActionsMarkup` or `dismissButtonMarkup` show should we hide this Box?

Reply:
Yeah good call. I don't thingk it makes any difference visually but there's no need to render it. I'll adjust the markup 👍